src/ReCaptcha.php

```diff
@@ -260,8 +260,11 @@ public function isValid($value, $context = null)
         }
 
         $service = $this->getService();
+        if (array_key_exists($value, $context)) {
+            $value = $context[$value];
```

Comment:
Instead of overriding `$value`, do use an `} else {` and call `$res = $service->verify(...)` with a different parameters. This avoids having a semantic change in the meaning of `$value`